hkm/templates/hkm/snippets/nav_basket_counter.html

```diff
@@ -4,16 +4,16 @@
     {% if request.basket.lines %}
     <div class="cart_text">{% trans 'Shopping cart' %}</div>
     <div class="cart_icons">
-      <img class="cart_svg" src="/static/hkm/svg/ShoppingCart.svg" alt="shopping">
+      <img class="cart_svg" src="/static/hkm/svg/ShoppingCart.svg" alt="{% trans 'Shopping cart' %}">
```

Comment:
Can you explain why this works? I'd assume that the screen reader would read "Shopping cart Shopping cart", but as far as I tested it, it only reads it aloud once.

Reply:
If you change browser width to use mobile navigation it actually reads "Shopping cart Shopping cart". Nice catch!